README.md

```diff
@@ -13,7 +13,7 @@ A Julia package for probability distributions and associated functions. Particul
 * Moments (e.g mean, variance, skewness, and kurtosis), entropy, and other properties
 * Probability density/mass functions (pdf) and their logarithm (logpdf)
 * Moment generating functions and characteristic functions
-* Sampling from population or from a distribution
+* Sampling a population or a distribution
```

Comment:
I guess this change should be reverted as we're not sampling populations or distributions but drawing samples from a population or distribution:
```suggestion
* Sampling from a population or from a distribution
```